dashboard/src/actions/repos.ts

```diff
@@ -358,30 +348,6 @@ export function findPackageInRepo(
   };
 }
 
-export function fetchImagePullSecrets(
-  namespace: string,
-): ThunkAction<Promise<void>, IStoreState, null, AppReposAction> {
-  return async (dispatch, getState) => {
-    const {
-      clusters: { currentCluster },
-    } = getState();
-    try {
-      dispatch(requestImagePullSecrets(namespace));
-      // TODO(andresmgot): Create an endpoint for returning just the list of secret names
-      // to avoid listing all the secrets with protected information
```

Comment:
We are finally doing it :P

Reply:
Yeah, I was sad I couldn't close the related issue (because it was already closed, not sure why).